RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+This release adds compatibility with uvicron 0.17
```

Comment:
Omicron is really getting into ppl's mind 👀

Reply:
😂